webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -29,6 +31,11 @@ const Breadcrumb = ({
         } = item;
         const overrideTitle = active && titleReplacement;
         const itemTitle = overrideTitle || text || caption;
+        const inner = active ? (
+          <EllipsisWithTooltip>{itemTitle}</EllipsisWithTooltip>
+        ) : (
+          itemTitle
+        );
```

Comment:
the brackets indentation looks odd to me, is that  eslint auto fix for linting?

Reply:
Yes (produced by `npm run lint -- --fix`), I wouldn't change it otherwise.